__tests__/app.test.js

```diff
@@ -251,3 +251,73 @@ test("200: responds with newly created comment when additional not needed infoma
       expect(comment).toHaveProperty("author", "rogersop");
     });
 });
+
+describe("PATCH", () => {
+  test("200: updates votes for the given article by article id and responds with the updated article", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article).toEqual({
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          article_id: 3,
+          votes: 1,
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("400: responds with appropriate error message when article id isan  invalid thingy magigy " , () => {
```

Comment:
As much as I love this test description, make them as easy to read / understand as possible